lib/ex_doc.ex

```diff
@@ -7,23 +7,24 @@ defmodule ExDoc do
     @moduledoc """
     Configuration structure that holds all the available options for ExDoc
 
-    You can find more details about this options in the `ExDoc.CLI` module.
+    You can find more details about these options in the `ExDoc.CLI` module.
     """
+
     defstruct [
       assets: nil,
       canonical: nil,
       deps: [],
       extra_section: nil,
       extras: [],
       filter_prefix: nil,
-      formatter: "html",
+      formatter: Mix.Project.config[:default][:formatter],
```

Comment:
This is the same issue. It is bad practice to invoke anything from Mix in your `lib` code because Mix is a build tool and it may not be available after the software is built. Even though you are using it only at compile time, doing this here means it is very easy for someone to accidentally depend on this at runtime.

What is this PR trying to achieve? Are the values being duplicated somewhere?